### DISCUSSION TITLE
TabularModel.fit( ) creates a directory `saved_models` in the project folder which stores `.ckpt`

### DISCUSSION THREAD

**Discussion by pranavvp16** [manujosephv/pytorch_tabular#273] [Q&A] [answered]
I want to know if there any is way to avoid this behaviour . I don't want the tabular_model.fit( ) to create the saved_models directory by default in my project folder

**Comment by manujosephv** [comment, marked as answer]
I thought I had answered this. Maybe I missed.
So by default, PyTorch Tabular enables a checkpoint save callback, which saves to `saved_models`

You can configure this through the `TrainerConfig` with the parameter `checkpoints_path`
